### PR TITLE
Update ranges.md

### PR DESCRIPTION
**Pull request by elect86**
Added `until` sample

comments made cleaner

### CODE REVIEW

File: pages/docs/reference/ranges.md [hunk 1]
```diff
@@ -11,9 +11,12 @@ Range expressions are formed with `rangeTo` functions that have the operator for
 Range is defined for any comparable type, but for integral primitive types it has an optimized implementation. Here are some examples of using ranges
 
 ``` kotlin
-if (i in 1..10) { // equivalent of 1 <= i && i <= 10
+if (i in 1..10)  // equivalent of i in [1, 10], 10 is included
     println(i)
-}
```
Comment (cypressious) [inline pages/docs/reference/ranges.md]: Please don't remove the curly braces.

File: pages/docs/reference/ranges.md [hunk 2]
```diff
@@ -11,9 +11,12 @@ Range expressions are formed with `rangeTo` functions that have the operator for
 Range is defined for any comparable type, but for integral primitive types it has an optimized implementation. Here are some examples of using ranges
 
 ``` kotlin
-if (i in 1..10) { // equivalent of 1 <= i && i <= 10
+if (i in 1..10)  // equivalent of i in [1, 10], 10 is included
```
Comment (cypressious) [inline pages/docs/reference/ranges.md]: The word "equivalent" implies this is real Kotlin syntax which it isn't. Maybe plain english would be more appropriate.

File: pages/docs/reference/ranges.md [hunk 3]
```diff
@@ -11,9 +11,12 @@ Range expressions are formed with `rangeTo` functions that have the operator for
...
     println(i)
-}
+    
+if (i in 1 until 10)  // equivalent of i in [1, 10), 10 is excluded
```
Comment (cypressious) [inline pages/docs/reference/ranges.md]: Same here with the word "equivalent".

File: pages/docs/reference/ranges.md (hunk 2 above)
Comment (cypressious) [inline pages/docs/reference/ranges.md]: Previously, it was valid Kotlin syntax, too.

Comment (elect86) [inline pages/docs/reference/ranges.md]: Ok, I removed `equivalent`

**Comment by yole** [comment]
Added a paragraph mentioning this function in a more appropriate place.

**Pull request closed without merging**